sdcm/tester.py

```diff
@@ -219,6 +220,14 @@ def get_duration(self, duration):
             duration = self.test_duration
         return duration * 60 + 600
 
+    def update_certificates(self):
+        try:
```

Comment:
raise `Exception` instead of calling to `self.fail` (it used to fail a test, setUp phase is not a test and in future will be refactored to run as an independent piece of functionality)

Reply:
Fixed.